test_util/cluster_api.py

```diff
@@ -396,7 +406,7 @@ def deploy_marathon_app(self, app_definition, timeout=120, check_health=True, ig
             applications. I.E:
                 [Endpoint(host='172.17.10.202', port=10464), Endpoint(host='172.17.10.201', port=1630)]
         """
-        r = self.post('/marathon/v2/apps', app_definition, headers=self._marathon_req_headers())
+        r = self.post('/marathon/v2/apps', json=app_definition, headers=self._marathon_req_headers())
```

Comment:
future thought: Ideally eventually it might be nice to move this to a `Marathon` API wrapper that we shared with the DC/OS CLI to have native Marathon API calling. Have to balance that a little bit with "want to make sure the Marathon API doesn't change unexpectedly"